packages/safe-tools-client/app/components/token-to-usd/index.gts

```diff
@@ -23,22 +27,30 @@ export default class TokenToUsd extends Component<Signature> {
 
   constructor(owner: unknown, args: Args) {
     super(owner, args);
-
     this.updateInterval = setInterval(() => {
       taskFor(this.tokenToUsdService.updateUsdConverter).perform(args.tokenAddress); 
     }, INTERVAL);
   }
   
   get usdAmount() {
-    return this.tokenToUsdService.toUsd(this.args.tokenAddress, BigNumber.from(this.args.tokenAmount))?.toString();
+    return this.tokenToUsdService.toUsd(this.args.tokenAddress, BigNumber.from(this.args.tokenAmount));
   }
 
   willDestroy() {
     clearInterval(this.updateInterval);
   }
 
   <template>
-    {{this.usdAmount}}
+    {{#if this.usdAmount}}
+      $ {{(weiToDecimal this.usdAmount @tokenDecimals)}} USD
+    {{else}}
+      <div class="token-to-usd-loading">
+        <BoxelLoadingIndicator class="token-to-usd-loading-indicator" @color="var(--boxel-purple-400)" />
+        <div class="network-connect-modal__waiting-status">
+          Converting to USD...
```

Comment:
@jurgenwerk Added the loading icon, maybe you have a better description for this.

Reply:
![Screen Shot 2023-01-13 at 16 05 28](https://user-images.githubusercontent.com/12637010/212282027-42a0fe1a-0e93-4b61-a9a3-57fd747fa46d.png)

Reply:
I think text is good but drop the spinner. To make it consistent with the loading indicator here:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/273660/212294029-b1a5c45d-1b37-4982-bcec-8e6a5584e249.png">


https://github.com/cardstack/cardstack/pull/3547#issuecomment-1380130274

Reply:
Updated